### PR TITLE
Request file when Rise Cache is not running

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. I decided to use the same HTTP status code check as what is being used for Rise Cache and not explicitly look for 403 or 404 errors. Let me know if that doesn't make sense. Thx.

**Comment by stulees** [comment]
@donnapep I think that's okay. I'm wondering though if you're going to have to add logic for a GET request fallback in here though to account for CAP, like we had to do in `rise-storage`.

**Comment by donnapep** [comment]
@stulees Yeah, perhaps I should. Not for CAP though since I'm not requesting anything from Rise Cache, but if the third party server doesn't support HEAD requests then that might be a problem.

File: dist/rise-cache.js [hunk 1]
```diff
@@ -49,53 +49,47 @@ RiseVision.Common.RiseCache = (function () {
...
-          if (status >= 200 && status < 300) {
-            callback(request);
-          } else {
-            callback(request, new Error("The request failed with status code: " + status));
-          }
-        });
-
-        xhr.send();
-
+        makeRequest("GET", url);
```
Comment (rodrigopavezi) [inline dist/rise-cache.js]: @donnapep I was think, should this also try a HEAD request as out rise cache accepts it? cheers

Comment (donnapep) [inline dist/rise-cache.js]: @stulees Yeah, that's a good idea as well. @stulees You OK with me changing that GET to a HEAD for Rise Cache, and then doing GET as a fallback instead? Thx.

Comment (stulees) [inline dist/rise-cache.js]: @donnapep Yeah I think that's a good call, thanks

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review latest. Thx.

**Comment by stulees** [comment]
@donnapep :+1:

**Pull request merged**